setup.py

```diff
@@ -33,6 +33,7 @@ def test_suite():
         'click',
         'pyyaml',
         'xmltodict',
+        'pytest',
```

Comment:
As pytest is only used for testing I am not sure if we should make it a general requirement. For people who only use this tool and don't want to develop anything this package would be installed even if it's never used. For those who want to contribute, the installation is prescribed in the [contributing.md](https://github.com/spdx/tools-python/blob/main/CONTRIBUTING.md#how-to-run-tests).

Reply:
Good point, moved it to test dependencies